franka_description/robots/inertial.yaml

```diff
@@ -0,0 +1,137 @@
+# This file does not contain official inertial properties
```

Comment:
The extraction of the dynamic parameters into their own yaml file is pretty nice. I like that a lot and didn't know such a thing was possible in XACRO. However it fails in the Melodic CI. Can you have look please?

Reply:
I think you just use an outdated version of xacro in your Melodic docker container. 
The Dockerfiles are missing an `apt-get upgrade`.

Reply:
Also `apt-get upgrade` is considered bad-practice in Dockerfiles AFAIR, because it would destroy reproducibility of docker images

Reply:
I leave how you interpret and format my comments up to you. In my working environment I, as a PR author, feel responsible for my pull requests and am grateful for every help and feedback I get on them while not relying on others to fix the issues with it. But maybe this is different in the FOSS community 🤷🏻‍♂️